### PR TITLE
bump to play 2.1.2 and cleanup

### PR DESCRIPTION
**Pull request by jamesward**

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -26,7 +26,6 @@ object ApplicationBuild extends Build {
   )
 
   val main = play.Project(appName, appVersion, appDependencies).settings(
-    scalaVersion := "2.10.1"
```
Comment (jamesward) [inline project/Build.scala:29]: I took this out because it caused some funkyness with Akka transitive dependencies.

Comment (jsuereth) [inline project/Build.scala:29]: huh, why not 2.10.2?

**Comment by jsuereth** [comment]
LGTM

File: project/Build.scala (hunk 1 above)
Comment (jamesward) [inline project/Build.scala:29]: Here is the funkyness: https://groups.google.com/forum/#!searchin/play-framework/akka$202.10.2/play-framework/esgvgNcJfl8/lPbfvN1SqtAJ

I didn't dig further.  Maybe it's not an issue.

I guess we need to determine if we should always bump the templates to everything that is the latest.  Or stick with what Play depends by default.

**Pull request merged**